go/decisions.md

```diff
@@ -1065,7 +1068,7 @@ can get wrapped into an interface and thus become a non-nil value (see the
 func Bad() *os.PathError { /*...*/ }
 ```
 
-**Tip**: A function that takes a `context.Context` argument should usually
+**Tip**: A function that takes a [`context.Context`] argument should usually
```

Comment:
Should be "**Tip:**".

Reply:
That should be fixed now.

Reply:
this should prob be handled in the copy.bara.sky file rather than modified by hand